CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - added fade animations
 - fixed certain inputs skipping too many things
 - fixed a memory leak in the audio sampler
+- added a vsync option
```

Comment:
Let's move it a bit to the top to keep the changelog items sorted via their category.